Main

ishitarakchhit opened the pull request.

AkankshaAI reviewed (approved):
@ishitarakchhit Can you please add detail

ishitarakchhit commented:
I have added a file in scripts to print hello world in c++. I have also added a file with name Ishita.md in profiles as my description. Further, updated CONTRIBUTING.md to add few details about myself.
